test/ibmq/test_account_client_v2.py

```diff
@@ -395,3 +395,19 @@ def test_api_version(self, qe_token, qe_url):
         api = AuthClient(qe_token, qe_url)
         version = api.api_version()
         self.assertIsNotNone(version)
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_user_urls(self, qe_token, qe_url):
+        """Check the user urls of the QX API."""
+        api = AuthClient(qe_token, qe_url)
+        user_urls = api.user_urls()
+        self.assertIsNotNone(user_urls)
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_user_hubs(self, qe_token, qe_url):
+        """Check the user hubs of the QX API."""
+        api = AuthClient(qe_token, qe_url)
+        user_hubs = api.user_hubs()
+        self.assertIsNotNone(user_hubs)
```

Comment:
Same as above; it'd be nice to verify the returned data structure matches expected (as specified in docstring)